.github/ISSUE_TEMPLATE/jnigen.md

```diff
@@ -0,0 +1,5 @@
+---
+name: "package:jnigen"
```

Comment:
Do we also add one for `package:jni`? I'm not aware of anyone using that package by itself as of yet.